pkg/vm/vm.go

```diff
@@ -63,6 +63,15 @@ const (
 // SyscallHandler is a type for syscall handler.
 type SyscallHandler = func(*VM, uint32) error
 
+// OnExecHook is a type for a callback that is invoked
+// for each executed instruction
```

Comment:
s/for each executed instruction/after each executed instruction ?

The order is important.

Reply:
Done